lib/migrator/index.js

```diff
@@ -142,10 +144,22 @@ const migrate = async () => {
       deleteIfUnchanged(patternsToDeleteIfUnchanged)
     ])
 
-    await Promise.all([
+    const [...partTwoResults] = await Promise.all([
       await removeOldPatternIncludesFromSassFile(patternsToDeleteIfUnchanged, 'app/assets/sass/application.scss'),
       await deleteEmptyDirectories(directoriesToDeleteIfEmpty)
     ])
+
+    const results = [...partOneResults, ...partTwoResults].flat()
+
+    const failure = results.some(result => !result)
+
+    if (failure) {
+      console.log('\nThe script is unable to fully migrate your prototype to version 13. Contact the GOV.UK Prototype team for support.')
```

Comment:
Hmm, I think we probably want a line break between these two sentences, just to keep the number of columns down.

```suggestion
      console.log('\nThe script is unable to fully migrate your prototype to version 13.')
      console.log('\nContact the GOV.UK Prototype team for support.')
```